cmd/clef/datatypes.md

```diff
@@ -0,0 +1,220 @@
+## UI Client interface
+
+These data types are defined in the channel between clef and the UI
+
+### SignDataRequest
+
+SignDataRequest contains information about a pending request to sign some data.The data to be  signed can be of various types, defined by content-type. Clef has done mostof the work in canonicalizing and making sense of the data, and it's up to the UI to presentthe user with the contents of the `message`
+
```

Comment:
Spaces in the wrong places, pointed out by scared face emoji:

 some data.😱The data to be 😱 signed can be of various types, defined by content-type. Clef has done most😱of the work in canonicalizing and making sense of the data, and it's up to the UI to presentthe user with the contents of the `message`